pkg/lib/cockpit-po-plugin.js

```diff
@@ -111,8 +109,19 @@ function init(options) {
 function run(webpack_module, webpack_compilation) {
     const promises = [];
     config.subdirs.map(subdir =>
-        promises.push(...get_po_files().map(po_file =>
-            buildFile(po_file, subdir, webpack_module, webpack_compilation))));
+        promises.push(...get_po_files().map(f => {
+            if (webpack_compilation)
+                webpack_compilation.fileDependencies.add(f);
+            const lang = path.basename(f).slice(0, -3);
+            return Promise.all([
+                // Separate translations for the manifest.json file and normal pages
+                buildFile(f, subdir, webpack_module, webpack_compilation,
+                          `${subdir}/po.${lang}.js`, str => !str.includes('manifest.json')),
```

Comment:
`.endsWith('/manifest.json')` rather, no?

Reply:
Looking at the po files we do it correct:
```
po/cs.po:#: pkg/storaged/manifest.json:0
```

Reply:
then maybe add `/` and `:` to the string we match on?

I guess this is not super important...